### PR TITLE
[DAS-32] Add Mosaic To The Dashboard

### PR DESCRIPTION
**Pull request by Alderr**
## Purpose/Description:
Get mosaic working with our tests and general improvements. 

- Due to issues with new builders like Vite preferring ESM (ECMAScript modules) which are faster, I exported Mosaic code using ESM.
- The paradigm for exporting is now to explicitly name and export types and components.
- Some aliases were removed for better clarity on how we will import components.

## Jira Link:

[DAS-32](https://collagegroup.atlassian.net/browse/DAS-32)

## Test Plan:

-   **Test Scenario 1:** Check if Mosaic packages correctly

    -   **Steps:**
        1. Run `npm install` in the app directory
        2. Run `npm run rollup` and verify there are no errors when packaging the application
    -   **Expected Result:** Rollup should package with no errors. Should look like this. 
    
image

[DAS-32]: https://collagegroup.atlassian.net/browse/DAS-32?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/atoms/CompanyLogo/companyLogos.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const companyLogos = {
+  levi: {
+    src: '',
+    alt: '',
+  },
+};
+
+export default companyLogos;
```
Comment (Alderr) [inline src/components/atoms/CompanyLogo/companyLogos.ts:8]: I want to set a paradigm to only use default exports for files that export a singular thing.

**Comment by darkares23** [comment]
Are we going to do something with the warnings to?

```
(!) Unresolved dependencies
https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency
src/typings (imported by "dist/esm/components/atoms/Button/Button.d.ts")
```
Maybe this is a dependency with some path error or is not well imported

**Comment by Alderr** [comment]
> Are we going to do something with the warnings to?
> 
> ```
> (!) Unresolved dependencies
> https://rollupjs.org/troubleshooting/#warning-treating-module-as-external-dependency
> src/typings (imported by "dist/esm/components/atoms/Button/Button.d.ts")
> ```
> 
> Maybe this is a dependency with some path error or is not well imported

@darkares23 
In the future, can you directly comment on the code? It's easier to respond to. 

And yes, we will address it. It's related to absolute imports and my `typings` folder confusing the rollup package setup. If I use a relative path import, the warning goes away. I spent some time on it since I still want to use absolute imports but it's not worth spending too much time on it since the library works fine at the moment.

File: src/components/atoms/CompanyLogo/companyLogos.ts (hunk 1 above)
Comment (darkares23) [reply src/components/atoms/CompanyLogo/companyLogos.ts:8]: Agree, default exports for files that export a singular thing is a good practice that can help make the codebase more consistent

**Review by darkares23 — APPROVED** [review]
Thank you, Vernon. I have been reviewing the code, understanding it, and testing it, and so far, I haven't found any errors. It seems like a significant improvement to have changed the file exports. I really like the way the structure is divided into atoms and the use of Storybook. Well done!

**Review by diego20 — APPROVED** [review]
LGTM!

**Pull request merged**